src/data_store.rs

```diff
@@ -70,6 +128,39 @@ impl DataStore {
     }
 }
 
+// Store all provided exchange rates.
+// For every row it takes ~13 bytes (4 + 3 + 2 + 4), if we have 100 fiat currencies it adds 1300 bytes.
+// For 1000 payments it will add ~1 MB.
+fn insert_snapshot(
```

Comment:
not very idiomatic function name. Isn't it a snapshot of exchange rates? How about `snapshot_exchange_rates`? Or a shorter version would be `snap_rates`. Or any combination of this.

Reply:
Sure. Changed in [a761e80](https://github.com/getlipa/lipa-lightning-lib/pull/628/commits/a761e80dc06674ed86422f8537484ce3d8330912)